app/models/user.php

```diff
@@ -82,5 +81,148 @@ public function login(LoginInfo $login_info)
             throw new RecordNotFoundException('No Record Found');
         }
         Session::set('username', $this->username);
+        Session::set('id', $row['id']);
+    }
+
+    public static function getAll()
+    {
+        $user = array();
+        $db = DB::conn();
+        $rows = $db->rows("SELECT * FROM user WHERE id = ?", array(Session::get('id')));
+
+        foreach ($rows as $row) {
+            $user[] = new self($row);
+        }
+        return $user;
+    }
+
+    public static function getComments()
```

Comment:
getUserComments function will use user id of other users and it will be used to view other users profile while getAll function will use id of the current user. :bow: